core/src/main/java/org/jdbi/v3/core/statement/SqlStatements.java

```diff
@@ -158,7 +158,8 @@ public SqlStatements setTemplateEngine(TemplateEngine templateEngine) {
     }
 
     /**
-     * Sets the Caffeine cache used to avoid repeatedly parsing SQL statements.
+     * Sets the Caffeine cache used to avoid repeatedly parsing SQL statements.<br>
```

Comment:
yeah, but ... is it? :-)

Reply:
I added another sentence and for nicer formatting the `<br>` in between.
The concurrency behaviour of SqlStatements depends on the type of cache which can be set from outside therefore thread-safety is up to the user.